plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -456,6 +457,20 @@ private ClasspathEntry fixEntry(ClasspathEntry e) {
     return result.stream().map(LanguageServiceUtils::removePrefixUri).collect(Collectors.toList());
   }
 
+  /**
```

Comment:
JavaLanguageServerExtensionService is made for communication with the browser. That's one reason why the methods are synchronous. This method here is NOT for that purpose, so should be somewhere else.

Reply:
Got it